lms/djangoapps/teams/csv.py

```diff
@@ -154,6 +155,7 @@ def set_team_membership_from_csv(self, input_file):
                 return False
             row_dictionaries.append(row)
 
+        self.number_of_learners_assigned = len(row_dictionaries)
```

Comment:
thought:  have you considered using `enumerate()`  to track where we are in the `row_dictionaries`?
This would enable us to know which row an error occurred on.  
```
for row_num, row in enumerate(row_dictionaries):
     self.add_user_to_team(row)
     self. number_of_learners_assigned  = row_num
```

Reply:
Haven't considered enumerate. I am not quite following how that would change/improve existing behavior.

Reply:
Mostly against the rule of _reveals intent_. As named, it's describing what has actually been done, as opposed to how it's being used -- describing the _likely_ final state.  Which is why I'm not even thinking of it as a nit.

Reply:
In any case, we should definitely set this _after_ the chunk of code below that actually adds users to teams, right?  That's my main takeaway from Matt's suggestion.

Reply:
Sounds good.